elastic-agent-client.proto

```diff
@@ -292,12 +292,28 @@ message CheckinAgentInfo {
     bool snapshot = 3;
 }
 
+// Feature flags configurations.
+// Added on Elastic Agent v8.7.1.
+message Features {
+    FQDNFeature fqdn = 2;
+}
+
+// FQDN feature flag indicates to use FQDN for host.name instead of hostname.
+// Added on Elastic Agent v8.7.1.
+message FQDNFeature {
+    bool enabled = 1;
+}
+
 // A set of units and their expected states and configuration.
 message CheckinExpected {
     // Units is the expected units the component should be running.
     repeated UnitExpected units = 1;
     // Agent info is provided only on first CheckinExpected response to the component.
     CheckinAgentInfo agent_info = 2;
+
+    // Features are the expected feature flags configurations.
+    // Added on Elastic Agent v8.7.1.
+    Features features = 3;
```

Comment:
Lets add the `unit64 features_idx = 4;`to this expected states. Also add the `features_idx` to the observed state.